src/Service/Cases.php

```diff
@@ -28,12 +28,12 @@ protected function getApiBasePath(): string
      *
      * @throws GetOrganizedClientException
      */
-    public function FindCases(array $query)
+    public function FindCases(array $data)
```

Comment:
The parameter should not be called `$data`, but rather `$query`. “Data” is used for creating stuff and not for querying.

Reply:
Updated.